system_test/aest_sync_SUITE.erl

```diff
@@ -207,7 +208,7 @@ new_node_joins_network(Cfg) ->
     inject_spend_txs(old_node1, patron(), 5, 1, 100),
     inject_spend_txs(old_node2, patron(), 5, 6, 100),
 
-    wait_for_value({height, Length}, [old_node1, old_node2], Length * ?MINING_TIMEOUT, Cfg),
+    wait_for_value({height, Length + 1}, [old_node1, old_node2], Length * ?MINING_TIMEOUT, Cfg),
```

Comment:
What's the reasoning behind this `+ 1` here?

Reply:
Micro forks. If we stop and check just after the height has been reached, it may well be that a little micro-block is discarded and hence hashes are different. We use this as common trick.